packages/iov-keycontrol/src/keyring-entries/slip10.ts

```diff
@@ -68,6 +70,7 @@ export class Slip10KeyringEntry implements KeyringEntry {
     cls: Slip10KeyringEntryConstructor = Slip10KeyringEntry,
   ): Slip10KeyringEntry {
     const data: Slip10KeyringEntrySerialization = {
+      id: Slip10KeyringEntry.generateId(),
```

Comment:
ahh, here we generate, not in constructor.
a bit confusing at review... but i think it is correct

Reply:
Yeah, this is a bit unintuitive as we do construction via deserialization. One could clean this up by using a constructor that takes all the fields and have a separate deserializer.